Gemfile

```diff
@@ -16,7 +16,7 @@ gem "rake", ">= 11.1"
 # be loaded after loading the test library.
 gem "mocha", "~> 0.14", require: false
 
-gem "capybara", "~> 2.13.0"
+gem "capybara", "~> 2.13"
```

Comment:
This is the version that Rails tests against, separate concern from the version requirement we use for newly-generated apps.